PatientWebApplication/UserMicroserviceApi/Model/PatientUser.cs

```diff
@@ -0,0 +1,154 @@
+/***********************************************************************
+ * Module:  Patient.cs
+ * Author:  Tamara
+ * Purpose: Definition of the Class Patient.Patient
+ ***********************************************************************/
+
+using System.Collections.Generic;
+
+namespace UserMicroserviceApi.Model
+{
+    enum Priority
+    {
+        Doctor,
+        Date,
+        None
+    }
+
+    public class PatientUser : User
+    {
+        public string medicalIdNumber { get; set; }
+
+        public bool isVerified { get; set; }
+        public string allergie { get; set; }
+        public string city { get; set; }
+        public bool guest { get; set; }
+        public string email { get; set; }
+        public string password { get; set; }
+        public bool isRegisteredBySecretary { get; set; }
+        public virtual List<ModelNotification> notifications { get; set; }
+        public bool isMarried { get; set; }
+        public string bornIn { get; set; }
+        public string parentName { get; set; }
+        public string exLastname { get; set; }
+        public string gender { get; set; }
+        public string file { get; set; }
+
+        public bool isBlocked { get; set; }
```

Comment:
Change properties names to start with upper letter and apply in rest of code (remove 'this' from constructors).